### PR TITLE
disable log request tests

### PR DESCRIPTION
**Pull request by PradeepKiruvale**
## Proposed changes

Disable the failing log request system tests.

## Types of changes

What types of changes does your code introduce to `thin-edge.io`?
_Put an `x` in the boxes that apply_

- [x ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (general improvements like code refactoring that doesn't explicitly fix a bug or add any new functionality)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
<br/>

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [ ] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by makr11st** [comment]
I think there is better way to disable a test in PySys: 

```xml
	<skipped reason="failing intermitently"/>
```

**Comment by rina23q** [comment]
> I think there is better way to disable a test in PySys:
> 
> ```
> 	<skipped reason="failing intermitently"/>
> ```

I also agree to use XML to disable the test. However, the spelling is correct? I didn't know the word and searched, then it suggested me `intermittently`.

**Comment by rina23q** [comment]
I like how to write the disable reason here: https://github.com/thin-edge/thin-edge.io/blob/main/tests/PySys/tedge/call_tedge_-h_-V/pysystest.xml#L25

```
  <skipped reason="Disabled, see https://cumulocity.atlassian.net/browse/CIT-318"/>
```

**Comment by makr11st** [comment]
`intermittently` is the correct spelling and it was just a suggestion to highlight this method to disable. It should contain the ticket number, yes.

**Comment by PradeepKiruvale** [comment]
> I like how to write the disable reason here: https://github.com/thin-edge/thin-edge.io/blob/main/tests/PySys/tedge/call_tedge_-h_-V/pysystest.xml#L25
> 
> ```
>   <skipped reason="Disabled, see https://cumulocity.atlassian.net/browse/CIT-318"/>
> ```
Fixed

File: tests/PySys/log_request_end_to_end/log_request_error_messages/run.py [hunk 1]
```diff
@@ -20,8 +20,12 @@
 
 class LogRequestVerifySearchTextError(EnvironmentC8y):
     operation_id = None
+    logRequest = None
```
Comment (rina23q) [inline tests/PySys/log_request_end_to_end/log_request_error_messages/run.py:23]: Do we need this change?

File: tests/PySys/log_request_end_to_end/log_request_num_lines/run.py [hunk 2]
```diff
@@ -19,8 +19,12 @@
 
 class LogRequestVerifyNumberOfLines(EnvironmentC8y):
     operation_id = None
+    logRequest = None
```
Comment (rina23q) [inline tests/PySys/log_request_end_to_end/log_request_num_lines/run.py:22]: Do we need this change?

**Review by rina23q — APPROVED** [review]
Approved

**Pull request merged**